troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+sudo timedatectl
+               Local time: Tue 2020-03-10 19:10:02 UTC
+           Universal time: Tue 2020-03-10 19:10:02 UTC
+                 RTC time: Tue 2020-03-10 19:36:53
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: yes
+              NTP service: active
+          RTC in local TZ: no
+```
+
+
+# IPv6 Disconnected Install Troubleshooting
+
+In this document we will analyze common scenarios of errors/failures which can cause the cluster installation to fail. 
+
+## Common failures Basic checks before Installing
+
+Below are some precautions we should follow before even attempting the OCP4 Ipv6 deployments. 
+
+* [Ipv4-BM-IPI](https://github.com/openshift-kni/baremetal-deploy/blob/master/install-steps.md#preparing-the-provision-node-for-openshift-install) has some pre-requisites which will be applied fpr ipv6 deployments as well. 
+
+### The architecture for IPv6 Disconnected Install(Nodes can vary as per requirements) 
+* Jump Host which acts as a DNSMasq, Registry. 
+* Worker/Provisioning node 
+* 3 Master nodes. 
+
+Before starting the installation, we need to set up a DNSmasq conf file which will assign IPv6 IP's to all the nodes on `eno3` interface.A sample file is provided below.  
+
+* A Jump Host should have one interface say `eno1` on Ipv4(Public Network), `eno3` on **Private VLAN** this will be used to provide **IPv6 IP's on the isolated VLAN**. 
+
+```
+domain-needed
+bogus-priv
+strict-order
+#expand-hosts
+bind-dynamic
+dhcp-sequential-ip
+dhcp-authoritative
+dhcp-fqdn
+log-queries
+domain=<Your_Domain_Name>
+resolv-file=/etc/resolv.conf.upstream
+interface=eno3
+enable-ra
+server=<Dns_Server>
+
+dhcp-range=<Ipv6-Range-1>,<IPv6-Range-2>,<Network-Subnet> 
+dhcp-option=option6:dns-server,[Jump-Hosts-IPv6-IP]
+dhcp-option=option6:ntp-server,[::]   
+
+local=/<Your_FQDN>/
+
+#static addresses
+address=/.apps.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/api.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/ns1.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/registry.<Cluster-Name>.<Domain-Name>/**<IPv6-IP-Jump-Host>**
+
+#reserved dynamic addresses
+dhcp-host=<MAC-Address-eno3-Interface>,master-0.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-0]
+dhcp-host=<MAC-Address-eno3-Interface>,master-1.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-1]
+dhcp-host=<MAC-Address-eno3-Interface>,master-2.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-1]
+dhcp-host=<MAC-Address-eno3-Interface>,worker-0.<Cluster-Name>.<Domain-Name>,[IPv6-IP-worker-0]
+```
+
+A sample `etc/resolv.conf.upstream` file is as shown below
+
+```
+# Generated by NetworkManager
+search <Cluster-Name>.<Domain-Name>
+nameserver <IPv6-IP-Jump-Host>
+```
+
+A sample `/etc/hosts` file should be configured as shown
+
+```
+<IPv6-IP-Jump-Host> registry.<Cluster-Name>.<Domain-Name>
+```
+
+**Note:** After all the above file changes, make sure you can resolve all the above hosts using nslookup and dig commands.
+
+# Registry Issues
```

Comment:
These don't seem like issues but your telling me how to setup the registry.

Reply:
Removing the above section.